Riot/Modules/Room/RoomCoordinator.swift

```diff
@@ -443,181 +424,6 @@ final class RoomCoordinator: NSObject, RoomCoordinatorProtocol {
 //
 //        navigationRouter?.present(coordinator, animated: true)
 //        coordinator.start()
-=======
-    private func showLiveLocationViewer() {
```

Comment:
Should we keep changes coming from element and comment them ?

Reply:
I think it's a rebase issue. I will fix it.

Reply:
I compared the two `RoomCoordinator.swift` files (the one in this rebase and the one in Element 1.8.20), and the only differences I have are the commented code for the disabled features.